tpl/math/math.go

```diff
@@ -135,6 +148,17 @@ func (ns *Namespace) Round(x interface{}) (float64, error) {
 	return _round(xf), nil
 }
 
+// Sqrt returns the square root of a number.
+// NOTE: will return for NaN for negative values of a
+func (ns *Namespace) Sqrt(a interface{}) (float64, error) {
```

Comment:
This is present in the diff because I moved the sqrt function (alphabetical order) not because I changed it, but I will remove the comment and resubmit. Per your [comments here](https://github.com/gohugoio/hugo/pull/6948#discussion_r383500103) I will not add a test case.

Reply:
Ah, OK, thanks.